mautic/mautic
# change any occurrence of contact list to contact segments, changed li…

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed issues  |  no

## Description
This PR changes any occurrence of list to segment in the email bundle and changes the list icon to pie chart
## Steps to test this PR
- Apply PR
- See list email change to segment email and list icon change to pie chart
